### PR TITLE
Renaming work

### PR DESCRIPTION
**Pull request by whitej6**

### CODE REVIEW

File: docs/admin/uninstall.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ Here you will find any steps necessary to cleanly remove the App from your Nauto
 
 ## Database Cleanup
 
-Prior to removing the plugin from the `nautobot_config.py`, run the following command to roll back any migration specific to this plugin.
+Prior to removing the app from the `nautobot_config.py`, run the following command to roll back any migration specific to this app.
 
 ```shell
-nautobot-server migrate nautobot_plugin_secrets_providers zero
+nautobot-server migrate nautobot_secrets_providers zero
```
Comment (HanlinMiao) [inline docs/admin/uninstall.md:10]: Just trying to make sure if the command is supposed to be this.

```suggestion
nautobot-server migrate nautobot_app_secrets_providers zero
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ on: # yamllint disable-line rule:truthy rule:comments
   pull_request: ~
 
 env:
-  PLUGIN_NAME: "nautobot-plugin-secrets-providers"
```
Comment (gsnider2195) [inline .github/workflows/ci.yml:17]: ENV var is still `PLUGIN_NAME`?

File: pyproject.toml [hunk 3]
```diff
@@ -5,9 +5,9 @@ description = "Nautobot Secrets Providers App"
 authors = ["Network to Code, LLC <opensource@networktocode.com>"]
 license = "Apache-2.0"
 readme = "README.md"
-homepage = "https://github.com/nautobot/nautobot-plugin-secrets-providers"
-repository = "https://github.com/nautobot/nautobot-plugin-secrets-providers"
-keywords = ["nautobot", "nautobot-plugin"]
+homepage = "https://github.com/nautobot/nautobot-app-secrets-providers"
+repository = "https://github.com/nautobot/nautobot-app-secrets-providers"
+keywords = ["nautobot", "nautobot-plugin", "nautobot-app"]
```
Comment (HanlinMiao) [inline pyproject.toml:10]: Shall we remove "nautobot-plugin" from the keyword?

**Review by HanlinMiao — APPROVED** [review]

File: .github/workflows/upstream_testing.yml [hunk 4]
```diff
@@ -10,4 +10,4 @@ jobs:
     uses: "nautobot/nautobot/.github/workflows/plugin_upstream_testing_base.yml@develop"
     with:  # Below could potentially be collapsed into a single argument if a concrete relationship between both is enforced
       invoke_context_name: "NAUTOBOT_SECRETS_PROVIDERS"
-      plugin_name: "nautobot-plugin-secrets-providers"
```
Comment (gsnider2195) [inline .github/workflows/upstream_testing.yml:13]: I guess we need to update the upstream workflow before we can change the input variable to `app_name`.

File: docs/admin/uninstall.md (hunk 1 above)
Comment (whitej6) [reply docs/admin/uninstall.md:10]: The previous command was not valid and came from an issue with the old cookie.

File: pyproject.toml (hunk 3 above)
Comment (whitej6) [reply pyproject.toml:10]: Leaving for searchability.

**Review by gsnider2195 — APPROVED** [review]

**Pull request merged**